install/check-minimum-requirements.sh

```diff
@@ -7,8 +7,9 @@ function vergte () { printf "%s\n%s" $1 $2 | sort --version-sort --check=quiet -
 
 
 
-DOCKER_VERSION=$(docker version --format '{{.Server.Version}}')
-if [[ -z "$DOCKER_VERSION" ]]; then
+if [[ $(docker version --format '{{.Server.Version}}') ]]; then
+  DOCKER_VERSION=$(docker version --format '{{.Server.Version}}')
+else
```

Comment:
How about this to avoid repeating?

```suggestion
DOCKER_VERSION=$(docker version --format '{{.Server.Version}}' || echo '')
if [[ -z "$DOCKER_VERSION" ]]; then
```